### DISCUSSION TITLE
EddieHub Open Source Ambassador

### DISCUSSION THREAD

**Discussion by vinzvinci** [EddieHubCommunity/support#1779] [Ideas] [answered]
Lots of folks are requesting event support like to speak at their event but Eddie is busy sometimes or has a full schedule. I think it's better if there will be volunteers to speak at their event `(1 person is backup to answer audience question if ever the speaker can't answer it well, 1person is for public speaking)`. The team can be called **EddieHub Open Source Ambassador** 😄.  I heard many amazing folks participating in Public speaking, they can be DevRel.
 
What do you think about **EddieHub Open Source Ambassador** title? 

Perks:
- Cool certificate?
- The Open Source Ambassador can be seen at EddieHub org.?
- Recognition from EddieHub members and from Eddie Jaoude `(We can write them a good recommendation on LinkedIn)`.

Edited:

EddieHub Ambassador is up:
https://twitter.com/eddiejaoude/status/1480901597567889411

**Comment by kaiwalyakoparkar** [comment]
That's a really cool idea!!! I am in :)

**Reply by vinzvinci** [reply]
You are 100% fit for this role

**Reply by kaiwalyakoparkar** [reply]
Haha! Thanks. Let's wait till community's thoughts on this!

**Reply by verma-kunal** [reply]
Would definitely contribute to this initiative 😃

**Reply by ghost** [reply]
> You are 100% fit for this role

Completely True

**Reply by ghost** [reply]
> Would definitely contribute to this initiative 😃

Count me in for that

**Reply by LuffySama-Dev** [reply]
Count me in

**Comment by rohan-kulkarni-25** [comment]
That's Great But just I want to say something !! The idea is super cool but Perks. This will attract unwanted people to join us and they will join us just for a Cool certificate, Recognition from EddieHub members, and from Eddie Jaoude (We can write them a good recommendation on LinkedIn).   We have to do something different so that only those come who are interested to be even if nothing is given to them !!

**Reply by kaiwalyakoparkar** [reply]
I think we should not provide certificate for participation but if the person does extensive community work and volunteering! What do you think on this :thinking:

**Reply by vinzvinci** [reply]
How about... there is an expiry of the role like you are EddieHub Open Source Ambassador until you graduate from college or you are EddieHub Open Source Ambassador for 2years? so anyone can apply for this role. There should be `FAQ` and `qualifications` listed on the website.

**Reply by vinzvinci** [reply]
There will be 2 certs. 1 is recognition certificate that the person is part of the program then the certificate of completion will be given if the person completed the roles after 2 years? Something like that. And the recommendations will be provided after the completion.

Public speaking every weekend in the discord stages is the training ground for the folks.

**Reply by starlightknown** [reply]
Wouldn't 2 years be too much for now because we are just starting out and we don't know how they are going to perform. Plus new people should get chance too

**Reply by vinzvinci** [reply]
How about 6 months? Then after the completion of 6months, they can be called EddieHub Open Source Alumni and they will give testimonials on what they experience?

**Reply by kaiwalyakoparkar** [reply]
Yeah! We can then encourage these alumni to mentor new folks about the ambassadorship program.

**Reply by Vyvy-vi** [reply]
> There will be 2 certs. 1 is recognition certificate that the person is part of the program then the certificate of completion will be given if the person completed the roles after 2 years? Something like that. And the recommendations will be provided after the completion.

Q) Would linking a certification to the program, could it cause some form of issues?

**Reply by vinzvinci** [reply]
I don't see it as a problem, since we will get to know them first. We will know who will be fit for the program or we know the person who's applying 😉 
![image](https://user-images.githubusercontent.com/73097560/122071720-8ed16280-ce29-11eb-84b8-335f42564cdb.png)

**Reply by Vyvy-vi** [reply]
I had a small idea - What if we make the application as a community application? (Someone from TeamEddie writes someone else's application). This would potentially rule out the possibility of people that come to the program attracted by the perks, and can potentially get applicants that are seen by the community as people that add value?

**Reply by ghost** [reply]
> I had a small idea - What if we make the application as a community application? (Someone from TeamEddie writes someone else's application). This would potentially rule out the possibility of people that come to the program attracted by the perks, and can potentially get applicants that are seen by the community as people that add value?

Yep we can nominate people like GitHub Stars

**Reply by Akshu-on-github** [reply]
Hmm, I seem to remember some who have asked for nominations in exchange for nominations? Imho, best avoid such a scenario

**Reply by kaiwalyakoparkar** [reply]
@Akshu-on-github Yeah lol happened with me. :laughing:

**Comment by starlightknown** [comment]
I would suggest a selection category for this in order to avoid people who just come for perks or recognition. Some team Eddie members could select them by application?

**Reply by vinzvinci** [reply]
I agree, Team Eddie can help to select.

**Reply by kaiwalyakoparkar** [reply]
Yup, that would be a good idea. They just need to share their socials (GitHub, LinkedIn, Twitter), etc and we can filter them according to that. What do you think?

**Reply by starlightknown** [reply]
I think a proper application on how they would perform would do along with socials, we could ask them to write answers in a short way or a video? speaker selections are done in this way too as I saw

**Reply by kaiwalyakoparkar** [reply]
@starlightknown Hmm. Great point. We can do the application process somewhat like MLSA. (Video can be optional as many folks become camera consious (Including me too :sweat_smile: ))

**Reply by vinzvinci** [reply]
Good idea @starlightknown. In the application form, they will put their socials like GitHub, Twitter, LinkedIn, and some important information we need to know more about them. And in the video application, there will be provided 3-5 questions that they need to answer like 'Why they want to become Open Source Ambassador' and so on. But we need to provide qualifications like `'who to apply?'` and some FAQs before they apply `(this shown on the website)`.

**Reply by vinzvinci** [reply]
Hmm in MLSA form, they are still required to give one video @kaiwalyakoparkar to answer 1 question. Okay, we can limit the question for that concern? Other questions can be answered in written form.

**Reply by Akshu-on-github** [reply]
Maybe we could set some pre-requisites?

Say, should be an active member for more than 2 to 3 months
(Those applying for Campus Experts have to have the Student Pack for at least 6 months)

**Reply by ghost** [reply]
> Say, should be an active member for more than 2 to 3 months

Sad I'll need to wait for two months then 😭

**Reply by Akshu-on-github** [reply]
Well, it's just a suggestion - if done right, this could be something that stays with us for a long time, right?
And we could use the first few months to get a feel of the individual, and allow the individual to get used to the community and what we stand for

**Reply by kaiwalyakoparkar** [reply]
> Maybe we could set some pre-requisites?
> 
> Say, should be an active member for more than 2 to 3 months
> (Those applying for Campus Experts have to have the Student Pack for at least 6 months)

I guess we can help new comers start right from the beginning

**Reply by Akshu-on-github** [reply]
> I guess we can help new comers start right from the beginning

I get where you're coming from - but we'll need to onboard people who have shown the ability to commit. 6 months is a long time, after all!

We can always give newcomers tips, tricks and training during our weekly community calls

**Reply by ghost** [reply]
> I agree, Team Eddie can help to select.

Does it mean that members of Team Eddie will not be able to become an Open Source Ambassador?

**Reply by ghost** [reply]
> Yup, that would be a good idea. They just need to share their socials (GitHub, LinkedIn, Twitter), etc and we can filter them according to that. What do you think?

People can create new identities on the internet in 10 minutes

**Reply by ghost** [reply]
> @starlightknown Hmm. Great point. We can do the application process somewhat like MLSA. (Video can be optional as many folks become camera consious (Including me too 😅 ))

Plus 1, I will be more comfortable with audio only

**Reply by ghost** [reply]
> Good idea @starlightknown. In the application form, they will put their socials like GitHub, Twitter, LinkedIn, and some important information we need to know more about them. And in the video application, there will be provided 3-5 questions that they need to answer like 'Why they want to become Open Source Ambassador' and so on. But we need to provide qualifications like `'who to apply?'` and some FAQs before they apply `(this shown on the website)`.

Yep But my concern is, will that video be available only to the folks (including our decision team) we select? Like If someone wants to keep their video only available to >= team eddie members. [Asking this because I want to keep it that way 😅 ]

**Reply by akshitadixit** [reply]
> @starlightknown Hmm. Great point. We can do the application process somewhat like MLSA. (Video can be optional as many folks become camera consious (Including me too sweat_smile ))

But the main purpose for these ambassadors would be to speak at events when Eddie can't and being camera conscious doesn't help, I mean what does everyone else think?

**Reply by Akshu-on-github** [reply]
Quite a few good points raised here!

Like @akshitadixit pointed out, this is an opportunity to overcome camera consciousness. Many speeches, workshops and conferences are live-streamed/recorded, published on YouTube and shared on various social media handles

Additionally, as @krishdevdb brought up, we can't verify a person's identity with just their social handles. Having video authentication will add another layer of identification.

And it would much easier to cross verify if they add value to a project if they submit a write up about 1 to 3 OS projects they actively contribute to

**Comment by rohan-kulkarni-25** [comment]
I think rather than inviting members we should give the perks , certificates,s or swags to present members who came to Eddiehub for collaboration, not for swags !! Those who will speak on stage as a good speakers for certain consistent days and keep on doing it ! will get something worth efforts from Eddie or eddiehub !!

**Reply by ghost** [reply]
In that perspective They will know it in the first call, where others will get a cert. Then they will do the same to get a cert

**Reply by kaiwalyakoparkar** [reply]
I think those who have joined EddieHub don't really care about swags (My personal opinion) they focus more on collaboration. But inviting new folks will encourage them to collaborate. Even if not all, few folks getting interested in collaboration will satisfy our intention. What do you think?

**Reply by kaiwalyakoparkar** [reply]
> In that perspective They will know it in the first call, where others will get a cert. Then they will do the same to get a cert

Yeah! I agree with that. We really don't want to bribe them to speak. We want them to feel speakable!

**Reply by vinzvinci** [reply]
True, a lot of us prefer collaboration. Collaboration makes real hubber happy, second is swag. SWAG is a bonus point. 
`Hubber = EddieHub members`

**Reply by Akshu-on-github** [reply]
Personally, I think the speaking experience would be a huge plus point in the skills section of an individual's resume - an opportunity like that is more than enough for me!

**Reply by verma-kunal** [reply]
Totally agree with all the awesome points here!👍

**Reply by kaiwalyakoparkar** [reply]
> Personally, I think the speaking experience would be a huge plus point in the skills section of an individual's resume - an opportunity like that is more than enough for me!

Yup I feel the same

**Reply by Vyvy-vi** [reply]
> I think those who have joined EddieHub don't really care about swags (My personal opinion) they focus more on collaboration. But inviting new folks will encourage them to collaborate. Even if not all, few folks getting interested in collaboration will satisfy our intention. What do you think?

Ideally, that is what the community that stays is like, but we get some people that are not-value-aligned, every now-and-then. 
Alas, people have differences :/

**Reply by ghost** [reply]
> True, a lot of us prefer collaboration. Collaboration makes real hubber happy, second is swag. SWAG is a bonus point.
> `Hubber = EddieHub members`

💯 True

**Comment by Akshu-on-github** [comment]
Ooh, this sounds amazing! I'd definitely apply for it 😜

**Reply by verma-kunal** [reply]
yeah! me too 😃

**Reply by ghost** [reply]
Count me in aswell

**Reply by kaiwalyakoparkar** [reply]
Haha I guess we will be there to review the application and attend the session on behalf of (Eddie)/ EddieHub

**Reply by Vyvy-vi** [reply]
I'll be interested in how this turns out and who become our first batch of EddieHub ambassadors! 🚀

**Reply by ghost** [reply]
> I'll be interested in how this turns out and who become our first batch of EddieHub ambassadors! 🚀

That will be interesting. In my thought, we should keep the first batch only for old and trusted members. Those who already added a lot of value to our community, like Felix, Kaiwalya, you, Kunal, Ishan...

**Reply by Vyvy-vi** [reply]
Idk, I'd say let Eddie and the community decide :)

**Comment by ItsMeIshan** [comment]
EddieHub Open Source Ambassador this title rocks!! but how will we participate in this? or this will happen during our normal public speaking sessions?

**Reply by vinzvinci** [reply]
We're still discussing it. Wait for the announcement.

**Reply by kaiwalyakoparkar** [reply]
@ItsMeIshan Discussion is going on. Stay tuned for the announcement :wink:

**Reply by ghost** [reply]
We will tell you (and all hubbers [including me]) once it is decided!

**Comment by akshitadixit** [comment]
Everything sounds amazing, just that why not we remove the perks? 
* Getting to speak at such events is a perk in itself
* Even better exposure is yet another perk
* Will help in easing the selection process and filter out unmotivated candidates who otherwise would've come in for certs only.

**Reply by kaiwalyakoparkar** [reply]
@akshitadixit Umm you might be correct but giving perks will also ensure that they were officially selected!

**Reply by akshitadixit** [reply]
To ensure "officially selected" we can still have the second point from @vinzvinci 's suggestion

**Reply by kaiwalyakoparkar** [reply]
> To ensure "officially selected" we can still have the second point from @vinzvinci 's suggestion

Umm, you have a point. But the swags are like our way of congratulating their hard work and they would be getting it on reaching certain milestonesonly.

**Reply by vinzvinci** [reply]
Let me add this here:
![image](https://user-images.githubusercontent.com/73097560/120731764-d6491c00-c516-11eb-9bfd-52045ae5a0c7.png)

**Comment by vinzvinci** [comment]
Application form will open in month of february and august only. What do you think about it?

**Reply by verma-kunal** [reply]
How about we make that quarterly, like MLSA ? Would that be too much of work then, i suppose🤔

**Reply by Akshu-on-github** [reply]
If a community member is an ambassador for a 6 month window, I'd say it's a reasonable time frame 👍

**Reply by Akshu-on-github** [reply]
And we could always shorten it after we get more volunteers on board?

**Reply by kaiwalyakoparkar** [reply]
> And we could always shorten it after we get more volunteers on board?

I agree

**Reply by ghost** [reply]
What I think is that it should be 3months. Cause we have a lot of members in EddieHub already 🙂

**Comment by kaiwalyakoparkar** [comment]
I would love to join Eddie in the Hackathon mentoring sessions which Eddie would be attending in future :wink:

**Comment by vinzvinci** [comment]
Hmm, any suggestion on what to add on the website?
Ex.
- Embedded YouTube video about the program
- FAQs
- Link to the form (or maybe there's one page for the form like on MLSA?)

Start giving your ideas below

**Reply by Akshu-on-github** [reply]
Would the prerequisites fall under the FAQ? If not, I believe we'll need a section for that

**Reply by vinzvinci** [reply]
I think it is best to add that on FAQ too @Akshu-on-github

**Reply by Vyvy-vi** [reply]
I'm not sure if we should set up a site, **before** we decide on how to organize, plan and execute the program

**Reply by ghost** [reply]
+♾️, First let us organize, plan and execute

**Comment by praveenscience** [comment]
I am in for this too! This is super cool, can get all my University of Leicester kids here! 🔥

**Comment by Vyvy-vi** [comment]
A few thoughts and ideas about how we could plan on starting such a program-

[ **This is just a suggestion/discussion/proposal, and not how we should or would be doing the Ambassador program.** I would love to get criticism, comments, insights, questions, discussion on this so that in the process we can have a discussion about these things and come out with a better plan, which we can then discuss more on, till it fits the needs of the program :) ]

## Making Application-
In order to drive out self-gain and the apparent spam that might get caused by people that just want the title/perks attached to the program, perhaps we can set up a system where someone from the community, who has been a contributor for some time(6 months or so, and having Team-Eddie role, along with having interacted to some extent on our github repo) can suggest a person as an applicant in a github discussion post, rather than having people directly apply. (perhaps we can even set a limit of 3-4 nominations per selection cycle)

The benefit of this process might trim down applications to people that are actively contributing or getting noticed by the community.

## Selection Process-
Eddie probably doesn't have a lot of time, and thus might not be able to go through all the applications.
A potential solution might be setting some sort of selection steps(somewhat like this)-
- Get nominated for being an applicant by a TeamEddie member selection(from the above application process)
- Asking the nominated member if they would like to become an ambassador
- Get background-checked (the community could do this, however IMHO it would be better if community mods, maintainers, and a few more members(carefully selected) decide on this, in order to prevent some form of name-calling or issues like that. We could also have a small arrangement where if anyone would like to negate the nomination, they could contact the mods/members participating in this check, where this could be taken into consideration)
- Eligibility check (a couple of criteria, that I'm pulling out from the top of my head. Ideally we should discuss the criteria and decide on them, esp. after Eddie's consideration and opinion):
  - OSS-Community activity: Must have contributed to the opensource community (not specifically ours), to a considerable extent, which adds value.
  - Activity in our community: Should have been a member, long enough to work with EddieHub(possibly the ones that have attended/hosted a few public-speaking calls, or come on stream a few times, or have contributed to our projects to some commendable extent or helped out people in the community.) Now, yah this criteria might be a bit weird, but IMO it would be best if the selection process also considers, if the applicant does know "what EddieHub is about", and what are our values.
  - Proficiency: At the very least, the applicant should have a fair amount of experience with working Git and Github, probably also about how to maintain a project(nothing too fancy, just should have some info about that matter, so that they could considerably talk about maintaining projects and opensource software), should be commendable in one of the following - [software development, blogging, technical writing, contributing to opensource, helping or advising people in the community, public speaking... etc etc.]
  - Age: As much as we'd like to not exclude people based on age, an ambassador program should have some sort of min.-age-limit, in order to in-a-way comply with legalities etc. (I am not aware of the certain legalities, but at least 13-15+ should be a thing to be COPPA-compliant, and maybe 18+ (again, not sure). I think there must be some reason other programs seem to have some sort of age cap. ...Contact your nearest legal advisor for figuring this out :P)
- Motivations, Intentions check + some form of an interview(esp. Eddie should be the one having the final say in the selection, since EddieHub Ambassador ~= Eddie in a way recommending you, personally?)
This info kinda gives you some detail about the person, and why they want to be an ambassador (which would be useful info for checking if they are fit for the role)

## Ambassador-ship period-
In order to prevent a "bloat" of ambassadors, perhaps we can set up a cap on the number of ambassadors _proportional_ to the no. of people in the community(I think for no. of ppl in community, no. of people in github org could be a good metric).
[Example - 10 ambassadors per 1000 members]
We could keep a Quarterly or thrice in a year selection round, or maybe even a yearly selection 🤔 
We could also set up a cycling process, where we swap out half of the ambassadors every selection period or something like that.
Q) Why reduce the no. of ambassadors and set a cap on top of it?
This cap is much more like a soft-cap(but the general idea is to keep lesser ambassadors, in order to reduce the no. of people that Eddie/the community would have to check, and this could also save money on swags etc. if they get involved in the program).
Another reason would be that since it's fewer people, we can keep cyclic selections and have a good amount of the community exposed. There's also an issue with too many ambassadors, which is that handling the narrative, community values might be harder (at this stage for EddieHub, since we aren't MLSA or Github Campus Experts, and we don't have the resources they have).

## Terms(of sorts)-
- Social use of being a member of the program: Establishing clear grounds on this not being Work Experience, rather a Volunteer position. Also, events that the members of the program participate in, don't use the phrasing that "this is a collaboration with Eddiehub", _unless_ Eddie consents to this being a community collab.
- Behaviour: Regardless of the program, they would still be compliant to EddieHub Contributor Covenant/Code of Conduct, and answerable to Eddie, moderation and the community for actions that might violate that, within our events, or in places where they represent EddieHub. Graciously accept mistakes, and take constructive feedback, when they might be wrong with regard to some topic related to OpenSource.
- Representation: They may not officially make EddieHub guild/group/etc. on hackathons or events, without Eddie and the community consenting to it.
- some more stuff???? (idk lol :P) (help, ask a lawyer or some legal advisor)


Would love to hear everyone's thoughts on the above process, and if these are points that we could consider, debate and build on top of/learn from while deciding on the actual process?

**Reply by starlightknown** [reply]
Very well written but few points that came to my mind. If we don't do applications then finding new members would be a bit difficult, like few members who we already know are great but if we want some new folks to join us and increase the community then maybe we need a little something to get the attention of potential candidates. Also like you said 10 ambassadors is a good number but the people matching the category and might be interested are kinda less than that. Otherwise the points are great for eligibility, they might even help out beginners if they have knowledge you mentioned

**Reply by kaiwalyakoparkar** [reply]
Very well structured Vyvy. Karuna's points are good as well. :thinking:

**Reply by Vyvy-vi** [reply]
> Very well written but few points that came to my mind. If we don't do applications then finding new members would be a bit difficult, like few members who we already know are great but if we want some new folks to join us and increase the community then maybe we need a little something to get the attention of potential candidates. Also like you said 10 ambassadors is a good number but the people matching the category and might be interested are kinda less than that. Otherwise the points are great for eligibility, they might even help out beginners if they have knowledge you mentioned

Interesting consideration and I do agree, that this is a lot more aligned for community members, rather than new people. However, considering that this is an "OpenSource ambassador" program, probably we would want people that are able to represent EddieHub, since they have interacted with us, and know our mission and values.
(I'm not sure if the purpose of the ambassador program is to catalyse interaction from new folks, that might be motivated to join the program. However, adding that as a consideration could be a good idea. There's also a small case where external ambassador applications would become spammy since that might attract more people to get the perks, rather than contribute to the community. An example of this might be - the recent spam-instead-of-engagement concerns from Eddie with regard to the discord)

**Reply by vinzvinci** [reply]
`Let me add specific points, all of these roles are volunteer`
The **program manager** will handle them all but with the help of **opensource expert**, plus some selected **Team Eddie** can help to check their background `(social media and GitHub activity)` then the **program manager** and **opensource expert** will finalize the selection. After selection, we can team them up with the **opensource expert** who will guide them through the program. We can choose some ambassador and give them first activity like to speak at <a href="https://github.com/EddieHubCommunity/support/discussions/1710">https://github.com/EddieHubCommunity/support/discussions/1710</a> and also they can submit their incoming event at <a href="https://github.com/EddieHubCommunity/support/discussions/604">https://github.com/EddieHubCommunity/support/discussions/604</a>.

We can use this program structure:

![image](https://user-images.githubusercontent.com/73097560/122146145-1601f300-ce89-11eb-99e6-7d5965266a03.png)

**Reply by ghost** [reply]
Looks #toogood Vyvy-vi's points and Vincent's Diagram 👌🏻

**Reply by vinzvinci** [reply]
Since the program is new, there's no opensource expert. Maybe ambassador can become an opensource expert in the future, there will be selection too based on their performance, and if they want to continue to help. Hmm idk, I'm suggesting what's best for the program. I think 4-6 maximum opensource expert people if ever the ambassadors become a big group.

**Reply by kaiwalyakoparkar** [reply]
Yeah, you are right @vinzvinci but they won't be managing people while being ambassadors so is it a Good Idea to tag them "Open Source Experts" directly. Coz In the first iteration we would be looking at everything keenly but how can we ensure the same smoothness amongst the TBD and Ambassador (Of course we would be there) but still we need to think for something that would help us select the folks amongst the ambassadors to become TBD. What do you think?

**Reply by Akshu-on-github** [reply]
A few of my thoughts:

- I have some doubts about the nominations. I think it's a good way to _boost_ a person's application, but I'm not sure if it should be the _only_ way to get into the program? I don't want to think badly of anyone in the community, but there have been spammy DMs in the past...

- > Must have contributed to the opensource community (not specifically ours), to a considerable extent, which adds value

  > (...) have contributed to our projects to some commendable extent

  `Considerable extent` and `commendable extent` sounds subjective and needs refining - it's an interesting idea, but if a person adds value to a community in a way other than code, it would be difficult to quantify it 🤔

- >  I think there must be some reason other programs seem to have some sort of age cap

  Imho, it's because what the program offers stops being relevant after a point in time. GCE is restricted to students enrolled in a verified institute (regardless of whether the individual a school student, undergrad or postgrad) as its training is aimed towards those who engage in student communities.

  On the other hand, the Linux Foundation isn't restricted as such. I think it's because it is for anyone who wants to contribute to Linux? (@felixfaisal would probably be able to explain it better)

  As far as the one for our community goes, I think it would be okay to leave it open to everyone who is above 18. Depending on the country, I believe minors require parental permission and similar legal documents?

- I like the `Terms` section. We could expand it by explaining when to and not to use the `branding` repo. Also, we should keep an eye on where the EddieHub logo's being used and with what context

- > In order to prevent a "bloat" of ambassadors, perhaps we can set up a cap on the number of ambassadors proportional to the no. of people in the community

  Sounds reasonable like a soft cap - we should focus on quality over quantity. And if it's in high demand, we can always increase the quota in the next iteration

- And hey, kudos to @Vyvy-vi, @vinzvinci, @starlightknown and @kaiwalyakoparkar for their awesome points 🎉

**Reply by Vyvy-vi** [reply]
Really Great points @Akshu-on-github. Made me think more about some of the parts of this.
  
- > Considerable extent and commendable extent sounds subjective and needs refining - it's an interesting idea, but if a person adds value to a community in a way other than code, it would be difficult to quantify it 🤔

  I was purposefully vague here(:joy: ) since I didn't want to be the one saying X number of contributions/Y months of membership etc(since, those metrics kinda encourage people to chase the metric, than actually adding value). Also, I don't know what should be the ideal metric, nor should I be the one to quote what's a good metric(ideally, I think it would be best if we leave that up to Eddie, and then discuss on it with Eddie and the community). About other contributions than code, I think we could maybe quantify mainly 5 types of contributions - Media(making EddieHub centric content or art), Helping people(willingness to help people in the community, moderation, maintaining the projects at our org.), Text content(making docs, making guides, making discord/blog posts etc. that help the community, making legal docs), Collaboration(coming on livestreams, talking to the community, interacting in community calls and maybe hosting them, collaborating with other communities and eddiehub to host cross-community calls/events) and code(*and obviously, knowing how to use git and github to a moderate extent could be a requisite)

- > I have some doubts about the nominations. I think it's a good way to boost a person's application, but I'm not sure if it should be the only way to get into the program? I don't want to think badly of anyone in the community, but there have been spammy DMs in the past...

  The main purpose behind having a nomination-based entry is to reduce the perks-driven application.
  Now, yes this can encourage people to spam DMs for getting an application in. This is a problem, however it can be handled much like how we handle usual DM spamming issues. Perhaps we could set up an add-on workflow where you can make an application on a github discussion, and the application goes through only after some people in the community(ideally team-eddie/people that are value-driven) in-a-way boost it up. (though again, this might have issues when only the same people 
boost the applications, which might be the case when only a few active members boost the applications)

- > As far as the one for our community goes, I think it would be okay to leave it open to everyone who is above 18. Depending on the country, I believe minors require parental permission and similar legal documents?

  hmm, I think 18 would be reasonable. maybe we can dial down to 16, if we want more age inclusivity(*legal consideration needed here)

- > Sounds reasonable like a soft cap - we should focus on quality over quantity. And if it's in high demand, we can always increase the quota in the next iteration

  hmm, a soft cap would be helpful, also along with increasing, we could consider cycling the old ambassadors with new ones.

**Comment by SaraJaoude** [comment]
There are some great ideas in this discussion, thanks everyone! Just to let you know we are working on putting the program together and should hopefully have something out soon. :)

**Reply by vinzvinci** [reply]
Thanks for the update :) Looking forward to that :octocat:

**Reply by Akshu-on-github** [reply]
I'm looking forward to it 👀
Ah, now I'm excited! ✨

**Reply by verma-kunal** [reply]
> There are some great ideas in this discussion, thanks everyone! Just to let you know we are working on putting the program together and should hopefully have something out soon. :) 

Really looking forward to this 🙂

**Comment by Panquesito7** [comment]
Amazing! Didn't see this discussion until today but I am totally in 💯

**Comment by eddiejaoude** [comment, marked as answer]
Going to mark this as `answered` as the program has been lanched here http://eddiehub.org/ambassador-program

**Reply by AryanParashar24** [reply]
@eddiejaoude Hope this happens again as I would have been totally in if it would have occurred now🥹
